### DISCUSSION TITLE
Am I too stupid to use fullUrls Modifier?

### DISCUSSION THREAD

**Discussion by nelo-digitalagentur** [statamic/cms#5000] [Q&A] [answered]
Hi,

happy new year to all. I am trying to use the full_urls modifier in my antlers template like this ```{{ url | full_urls }}```

Sad but the the output always stays the relative url. I tried to get the bottom of it and ended up in believing the regular expression used in the modifier might be incomplete? Can anybody point me in the right direction what's going wrong here? 

See screenshot:

<img width="876" alt="Bildschirmfoto 2022-01-02 um 16 22 52" src="https://user-images.githubusercontent.com/3713906/147880490-e4905ca9-0614-4e74-b499-530ee8334f69.png">

Expected result: https://www.domain.test/magazin/aktiv-bleiben/test

Instead the result ist: /magazin/aktiv-bleiben/test

Thx and regards,

Andreas

**Comment by jackmcdade** [comment, marked as answer]
Try permalink instead!

**Comment by nelo-digitalagentur** [comment]
@jackmcdade works! thx! Any idea why full_urls does not work? Did I misunderstand what full_urls should do?

**Comment by jackmcdade** [comment]
It replaces all the relative urls in a longer string, like a content block. It probably SHOULD have worked in your case but I bet it assumes more test than just the url

**Comment by jasonvarga** [comment]
It converts URLs of links inside content from relative to absolute URLs. It doesn't just transform a single URL.

You need to have `<a href="...">` in the var you're modifying.

**Comment by nelo-digitalagentur** [comment]
Thx for the hint! Maybe the description in the annotation could be something like this then: 

"Replace ="/relative/urls" with ="https://domain.com/absolute/urls."

I guess the regex also captures ```src=""``` and ```rel=""``` etc.

PS: damn regex...! ;-)

**Comment by jackmcdade** [comment]
Updated the docs 👍
